x/incentives/keeper/distribute_test.go

```diff
@@ -189,9 +188,8 @@ func (s *KeeperTestSuite) TestDistributeToConcentratedLiquidityPools() {
 		tokensToAddToGauge sdk.Coins
 		gaugeStartTime     time.Time
 		gaugeCoins         sdk.Coins
-		poolType           poolmanagertypes.PoolType
 		lockExist          bool
-		authorizedUptimes  []time.Duration
+		isBalancerPool     bool
```

Comment:
there is no test case that utilizes this param and we should

Reply:
There is a test `TestDistribute` that has a variety of test cases for this.

I don't think we need to be packaging ByDuration test cases into this "NoLock" test case

Reply:
Cleaned up the test case to remove obsolete things: [f9a3d73](https://github.com/osmosis-labs/osmosis/pull/5459/commits/f9a3d73d8fe258ceef58b2f0685c071690ca8940)